### PR TITLE
Seperated supplemental resources-Part 9

### PR DESCRIPTION
**Pull request by raibove**
Moved additional resources to supplemental resources

Fixed Issue: #837

### CODE REVIEW

**Comment by raibove** [comment]
@alodahl, Role-in-tech file does not contain Lesson section and does not have any supplemental resources too.

File: personal-brand/personal-branding.md [hunk 1]
```diff
@@ -27,9 +27,8 @@
 
 ### Materials
 
-- [templates.netlify.com](http://templates.netlify.com/)
-- [codenewbie](https://www.codenewbie.org/)
 - [How to win friends and stand out from the crowd](https://pizza.netlify.com)
+- [Tutorial](https://blog.markgrowth.com/how-to-start-a-beautiful-personal-blog-using-medium-ca6d89b7e46)-Create blog on Medium.
```
Comment (alodahl) [inline personal-brand/personal-branding.md]: Can you help us by noting what these three changed resources are next to each link?

Comment (raibove) [reply personal-brand/personal-branding.md]: @alodahl, Shall I add description next to each link. I am not clear with what you meant by "Can you help us by noting what these three changed resources are next to each link?"

**Comment by alodahl** [comment]
Please put a description next to each resource, so apprentices know what they’ll find there.

**Comment by raibove** [comment]
Ok Will do it. What about Role-in-tech file @alodahl

**Comment by alodahl** [comment]
> Ok Will do it. What about Role-in-tech file @alodahl

Yes please!  All of the links need quick descriptions.

**Comment by raibove** [comment]
> Yes please! All of the links need quick descriptions. You can add a ### Supplemental Resources section if you need one. Roles in tech probably doesn't need a lesson section.

Ok, Added description to links.

File: learning-to-learn/learning-to-learn.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ https://www.youtube.com/watch?v=-71zdXCMU6A
 
 - [Learning to Learn (video walkthrough of slides)](https://drive.google.com/open?id=1R7cp0huJu5kiIIfDsraFbtmGbINHb2Xe)
 - [Learning to Learn (slides)](https://docs.google.com/presentation/d/1pMkBP32lra-O_BS6kOqj81IzLspN1UtKT64jqOXS9Kc/edit?usp=sharing)
-- [Learning How to Learn (video)](https://www.youtube.com/watch?v=O96fE1E-rf8)
+- [Learning How to Learn (video)](https://www.youtube.com/watch?v=O96fE1E-rf8) - A TEDx video by  Barbara Oakley, in which she shares how did she changed her mindset.
```
Comment (Kundan28) [inline learning-to-learn/learning-to-learn.md]: Remove one whitespace between `by` & `Barbara Oakley`.

File: learning-to-learn/learning-to-learn.md [hunk 3]
```diff
@@ -91,6 +91,6 @@ Participants write a self-reflective journal entry covering:
 
 ### Supplemental Resources
 
-- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0)
-- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/)
-- [The Power of Believing You Can Improve (video)](https://www.youtube.com/watch?v=_X0mgOOSpLU)
\ No newline at end of file
+- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems
+- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some usefull tips which will help you to learn
+- [The Power of Believing You Can Improve (video)](https://www.youtube.com/watch?v=_X0mgOOSpLU) - A TEDx video by Carol Dweck in which she explain how mindset can affect activity of brain
```
Comment (Kundan28) [inline learning-to-learn/learning-to-learn.md]: Add full stop after all the three descriptions.

**Comment by raibove** [comment]
> @raibove Two changes to be made.

Made the changes @Kundan28

File: learning-to-learn/learning-to-learn.md [hunk 4]
```diff
@@ -91,6 +91,6 @@ Participants write a self-reflective journal entry covering:
 
 ### Supplemental Resources
 
-- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems
-- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some usefull tips which will help you to learn
-- [The Power of Believing You Can Improve (video)](https://www.youtube.com/watch?v=_X0mgOOSpLU) - A TEDx video by Carol Dweck in which she explain how mindset can affect activity of brain 
+- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems.
+- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some usefull tips which will help you to learn.
```
Comment (Kundan28) [inline learning-to-learn/learning-to-learn.md]: usefull -> useful

Comment (raibove) [reply learning-to-learn/learning-to-learn.md]: @Kundan28 I made the change. Please review my PR

File: personal-brand/personal-branding.md [hunk 5]
```diff
@@ -69,3 +68,8 @@ Tweet out about your new blog on Twitter using the [#codenewbie](https://twitter
 ### Check for Understanding
 
 Send your first blog post to a peer and ask for comments and 2 ways you can improve your post or your site.
+
+### Supplemental Resources
+
+- [templates.netlify.com](http://templates.netlify.com/) - It consist of variety of templates which can be used for your website
+- [codenewbie](https://www.codenewbie.org/) - It is a community for new coders which provides various resources and has a slack group for their help.
```
Comment (alodahl) [inline personal-brand/personal-branding.md]: A community for new coders that provides various resources and help, including a slack group.

File: personal-brand/personal-branding.md [hunk 6]
```diff
@@ -69,3 +68,8 @@ Tweet out about your new blog on Twitter using the [#codenewbie](https://twitter
 ### Check for Understanding
 
 Send your first blog post to a peer and ask for comments and 2 ways you can improve your post or your site.
+
+### Supplemental Resources
+
+- [templates.netlify.com](http://templates.netlify.com/) - It consist of variety of templates which can be used for your website
```
Comment (alodahl) [inline personal-brand/personal-branding.md]: Lists a variety of website templates.

File: learning-to-learn/learning-to-learn.md [hunk 7]
```diff
@@ -91,3 +88,9 @@ Participants write a self-reflective journal entry covering:
 1) What they learned about themselves, 
 2) What from this lesson they felt challenged by,
 3) What from this lesson they want to practice.
+
+### Supplemental Resources
+
+- [Growth Mindset (video)](https://www.youtube.com/watch?v=EyIF5VUOJc0) - This is summary of a book Growth Mindset by Carol Dweck, which depends on her research how do kids tackle with difficult problems.
+- [Learning to Learn by Sarah Drasner](https://css-tricks.com/learning-to-learn/) - It contains some useful tips which will help you to learn.
+- [The Power of Believing You Can Improve (video)](https://www.youtube.com/watch?v=_X0mgOOSpLU) - A TEDx video by Carol Dweck in which she explain how mindset can affect activity of brain.
```
Comment (alodahl) [inline learning-to-learn/learning-to-learn.md]: explains how mindset can affect brain activity.

File: personal-brand/personal-branding.md (hunk 6 above)
Comment (raibove) [reply personal-brand/personal-branding.md]: @alodahl I have not understood what change am I supposed to do in this sentence.

Comment (alodahl) [reply personal-brand/personal-branding.md]: change :  "It consist of variety of templates which can be used for your website" -->

"Lists a variety of website templates."

**Pull request merged**